doc/lua_api.txt

```diff
@@ -3783,6 +3783,10 @@ This is basically a reference to a C++ `ServerActiveObject`
     * `0`...`1`: Overrides day-night ratio, controlling sunlight to a specific amount
     * `nil`: Disables override, defaulting to sunlight based on day-night cycle
 * `get_day_night_ratio()`: returns the ratio or nil if it isn't overridden
+* `set_time_offset(offset)`
+    * Adds offset to the time of the player 
+    e.g. TimeOfDay is 12:00 and the offset of the player is 1000 then the time of the player is 13:00
```

Comment:
It's clearer to use 12000, 1000, 13000 here. This form of time seems more familiar to modders.